Libraries/Image/Image.android.js

```diff
@@ -152,6 +152,7 @@ var Image = React.createClass({
 
   render: function() {
     var source = resolveAssetSource(this.props.source);
+    var src = resolveAssetSource(this.props.src);
```

Comment:
You don't need to call `resolveAssetSource`, this is just wasteful :) You can just check `if (this.props.src)`